tests/test_ir.py

```diff
@@ -220,11 +227,32 @@ def test_iteration_instance_cmp(self, ii_num, ii_literal):
         assert fcxy <= fcxy
         assert fcxy < fcx1y
 
+    def test_timed_access_regularity(self, ta_literal):
+        """
+        Test TimedAcces.{is_regular,is_irregular}
+        """
+        (tcxy_w0, tcxy_r0, tcx1y1_r1, tcx1y_r1, rev_tcxy_w0, rev_tcx1y1_r1,
+         tcyx_irr0, tcxx_irr1, tcxy_irr2) = ta_literal
+
+        # Regulars
+        assert tcxy_w0.is_regular and not tcxy_w0.is_irregular
+        assert tcxy_r0.is_regular and not tcxy_r0.is_irregular
+        assert tcx1y1_r1.is_regular and not tcx1y1_r1.is_irregular
+        assert tcx1y_r1.is_regular and not tcx1y_r1.is_irregular
+        assert rev_tcxy_w0.is_regular and not rev_tcxy_w0.is_irregular
+        assert rev_tcx1y1_r1.is_regular and not rev_tcx1y1_r1.is_irregular
+
+        # Irregulars
+        assert tcyx_irr0.is_irregular and not tcyx_irr0.is_regular
+        assert tcxx_irr1.is_irregular and not tcxx_irr1.is_regular
+        assert tcxy_irr2.is_irregular and not tcxy_irr2.is_regular
+
     def test_timed_access_distance(self, x, y, ta_literal):
         """
         Tests comparison of objects of type TimedAccess.
```

Comment:
Nitpicking: Test, cause I saw you changed the previous one ;-)

Reply:
and the next one..

Reply:
fixed this one.

There are others, yes -- but the complete clean up is outside the scope of this PR